lib/transport/abstract-transport.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var util = require("util");
+var EventEmitter = require('events');
+
+var AbstractTransport = function () {
+	if (this.constructor === AbstractTransport) {
+		throwBecauseAbstract();
```

Comment:
no worries, I've removed AbstractTransport from the exports but left it there for the built-in transports. I'm asking because i'm new to all this :) I tried to find something to check for basic transport functionalities in TransportProvider.